web_search_with_and/templates/web_assets_backend.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="assets_backend" name="Horanet backend sources" inherit_id="web.assets_backend">
+            <!-- Backend javascript sources for altering the search method-->
+            <xpath expr="//script[last()]" position="after">
+                <script src="/horanet_web_search_with_and/static/src/js/inherit_search_view.js"
```

Comment:
OK, to recap and avoid wasting your time:
In order to respect commit history, I can migrate from the branch 8.0 even if the module isn’t present in 9.0
I remove this PR to create a new one
Sorry for the inconvenience